examples/gcal_oo_or.ty

```diff
@@ -90,7 +91,16 @@ p.add(
         Learning rate for the lateral excitatory projection to V1."""),
 
     inh_lr=param.Number(default=0.3,bounds=(0.0,None),doc="""
-        Learning rate for the lateral inhibitory projection to V1."""))
+        Learning rate for the lateral inhibitory projection to V1."""),
+
+    gpu=param.Boolean(default=False,bounds=(0,1),doc="""
+        Sets whether the simulation should be run on a GPU"""))
+
+
+if p.gpu:
+    from topo.gpu.projection import GPUSparseCFProjection
+    from topo.gpu.sheet import compute_sparse_gpu_joint_norm_totals, GPUSettlingCFSheet
+
```

Comment:
There is indeed a lot of duplication that happened because initially there was no separate GPUSettlingCFSheet class and because not all of the sparse projections have their GPU equivalents. I agree that it's probably better if I abandon the changes in the examples folder and move the model currently used for testing as suggested.